src/App.js

```diff
@@ -21,23 +21,29 @@ class App extends Component {
     getMovies()
     .then(allMovies => this.setState({ movies: allMovies.movies }))
     .catch(err => {
-      this.setState({  err: 'Sorry, try again later.'});
```

Comment:
I think that we should consider if the message returned by the error is actually what we want to show to the user, or if we should put a different value here